src/components.d.ts

```diff
@@ -16,17 +16,21 @@ export namespace Components {
          */
         "audio": string;
         /**
-          * Stylesheet
+          * Optional custom Stylesheet to override defaults
          */
-        "css_url"?: string;
+        "cssUrl"?: string;
         /**
           * Language
          */
         "language": string;
+        /**
+          * Toggles the page scrolling from horizontal to vertical. Defaults to horizontal
+         */
+        "pageScrolling": string;
```

Comment:
Declare the type as a [union of literals](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#literal-types) to unleash TypeScript awesomeness:

```suggestion
        "pageScrolling": "horizontal" | "vertical";
```

Reply:
great idea done!